### ISSUE TITLE
Model cannot converge when training coco panoptic task

### ISSUE THREAD

**Issue by loliq** [facebookresearch/MaskFormer#23]
Hello, Thanks for your contribution！I want to train coco panoptic model, but i met a convergence problem.  I use the `maskformer_panoptic_R50_bs64_554k.yaml` as my config, but I only have 4 P40 GPU, so I adjust the learning rate to 0.0000125. Buy after 400 K iterations, the total loss is still around 8.0. Except for the learning rate and the number of GPUs, I did not modify other parameters. Does the coco panoptic model have to be trained with 64 GPUs?

**Comment by bowenc0221** [comment]
Since we are use ADAMW optimizer, the linear scaling rule does not apply anymore. I would suggest you try different learning rate. The model does not necessarily need to be trained with 64 GPUs.
